### ISSUE TITLE
README tutorial

### ISSUE THREAD

**Issue by rogersamso** [worlddynamics/Vensim2MTK.jl#34]
Hi,

The mini tutorial you provide in the README does not work. Your example reads:

```julia
using Vensim

filepath= "path/to/a/vensim/model/model.xmile"
filename= "MTKmodel.jl"
overwrite = false
Vensim2MTK(filepath, filename, overwrite) 
```

And it should read:

```julia
using Vensim2MTK

filepath= "path/to/a/vensim/model/model.xmile"
filename= "MTKmodel.jl"
overwrite = false
vensim2MTK(filepath, filename, overwrite) 
```

**Comment by natema** [comment]
Thanks for flagging that up!

**Comment by rogersamso** [comment]
Hi,

Note the lowercase in the `vensim2MTK` function (last line of the example)

**Comment by natema** [comment]
Ups..!

**Comment by natema** [comment]
Fixed by #36
